flask_app/issue_trackers.py

```diff
@@ -36,7 +36,7 @@ class JIRA(Tracker):
     def __init__(self, *, url, config):
         self._url = url
         config = json.loads(config)
-        self._jira = JIRAAPI(url, basic_auth=(config['username'], config['password']))
+        self._jira = JIRAAPI(url, basic_auth=(config['username'], config['password']), timeout=60)
```

Comment:
This is quite high for a service in our local network. I'd say 5 is a reasonable number